staging/src/k8s.io/kubectl/pkg/cmd/create/create_test.go

```diff
@@ -35,13 +37,18 @@ func TestExtraArgsFail(t *testing.T) {
 	defer f.Cleanup()
 
 	c := NewCmdCreate(f, genericclioptions.NewTestIOStreamsDiscard())
-	ioStreams, _, _, _ := genericclioptions.NewTestIOStreams()
-	options := NewCreateOptions(ioStreams)
-	if options.Complete(f, c, []string{"rc"}) == nil {
+	if validateCreateArgs(c, []string{"rc"}) == nil {
 		t.Errorf("unexpected non-error")
 	}
 }
 
+func validateCreateArgs(cmd *cobra.Command, args []string) error {
+	if len(args) != 0 {
```

Comment:
As far as I can see, there are several validations happening in `ToOptions`. Would you mind adding more test cases to cover them?

Reply:
I was looking at PR that got merged for split the flag but I couldn't find a good example , do you have any pointer where I can find a good example?

`TestExtraArgsFail` used to use `Complete` function which I removed so instead of removing the Test function, I just followed what `apply` command to test the `TestExtraArgsFail` https://github.com/kubernetes/kubernetes/blob/6bdda2da160043c5fcdfd25bbea9e1c1b804c9e5/staging/src/k8s.io/kubectl/pkg/cmd/apply/apply_test.go#L95